src/maintainer/knowledge_base.rst

```diff
@@ -486,9 +528,58 @@ following changes before the builds scripts run:
   is also included in ``$PYTHONPATH``.
 - A copy of all ``$PREFIX`` site-packages to ``$BUILD_PREFIX`` (except the compiled libraries).
 
-All in all, this results in a setup where ``conda-build`` can run a ``$BUILD_PREFIX``-architecture 
+All in all, this results in a setup where ``conda-build`` can run a ``$BUILD_PREFIX``-architecture
 ``python`` interpreter that can see the packages in ``$PREFIX`` (with the compiled bits provided by
-their corresponding counterparts in ``$BUILD_PREFIX``) and mimic that target architecture.
+their corresponding counterparts in ``$BUILD_PREFIX``) and sufficiently mimic that target
+architecture.
+
+.. _emulation:
+
+Emulated builds
+---------------
+
+When cross-compilation is not possible, one can resort to emulation. This is a technique that uses
+a virtual machine  (`QEMU <https://www.qemu.org/>`__) to emulate the target platform, which has a
+significant overhead. However, ``conda-build`` will see the target platform as native, so very
+little changes are usually needed in the recipe.
+
+To enable emulated builds, you must use the :ref:`provider` mapping in ``conda-forge.yml``.
+This key maps a ``build_platform`` to a ``provider`` that will be used to emulate the platform.
+``conda-smithy`` will know how to detect whether the provider supports that platform natively or
+requires emulation, and will adjust the appropriate CI steps to ensure that QEMU runs the process.
+Ensure changes are applied by :ref:`rerendering <dev_update_rerender>` the feedstock.
+
+Note that only Linux architectures are currently supported via emulation.
+
+.. warning::
+
+  Emulated builds are very slow and incur an additional strain on conda-forge CI resources.
+  Whenever possible, please consider cross-compilation instead. Only use emulated builds as a last
+  resort.
+
+Emulation examples
+^^^^^^^^^^^^^^^^^^
+
+Configure ``conda-forge.yml`` to emulate ``linux-ppc64le``, but use native runners for ``linux-64``
+and ``linux-aarch64``. This works because ``linux-ppc64le`` is not natively supported by Azure, so
+``conda-smithy`` will add QEMU steps to emulate it. However, ``linux-64`` and ``linux-aarch64`` are
+natively supported by Azure and Travis CI, respectively, so no emulation is needed.
+
+.. code-block:: yaml
+
+    provider:
+      linux_aarch64: travis
+      linux_ppc64le: azure
+      linux_64: azure
+
+Use this variables in your builds scripts if needed:
+
+- ``CONDA_BUILD_CROSS_COMPILATION``: Set to ``1`` when build and host platforms differ. This can
+  mean your are cross-compiling or emulating.
+- ``CROSSCOMPILING_EMULATOR``: Path to the ``qemu`` binary for the host platform.
```

Comment:
This emulator is for cross compiling while the text around it has been for native compilation (full OS emulation is still native compilation).
This emulator is used when cross compiling and is used for running the tests after compiling is done. Or for when running some executables by the build system.

Reply:
Ohh I see, ok ok. I'll remove it from here then. Thanks!